manager/assets/modext/widgets/security/modx.panel.access.policy.template.js

```diff
@@ -129,6 +125,7 @@ Ext.extend(MODx.panel.AccessPolicyTemplate,MODx.FormPanel,{
         var r = this.config.record;
 
         this.getForm().setValues(r);
+        Ext.getCmp('modx-policy-template-header').html = Ext.util.Format.htmlEncode(r.name);
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(r.name));`